tests/cases/setup_tdereplication/playbook.yml

```diff
@@ -0,0 +1,30 @@
+---
+- hosts: all
+  name: Postgres deployment playbook
+  become: true
+  gather_facts: true
+  any_errors_fatal: true
+
+  collections:
+    - edb_devops.edb_postgres
+
+  pre_tasks:
+    - name: Initialize the user defined variables
+      set_fact:
+        disable_logging: false
+        edb_enable_tde: true
+        edb_master_key_location: "/etc/edb/keyloc"
```

Comment:
Will these values result in a successful test? There is no test script so I'm assuming it hasn't been tested yet?